server/stree/dump.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stree
+
+import (
+	"fmt"
+	"io"
+	"strings"
+)
+
+// TODO(dlc) - This I used alot for debugging and testing. Probably should be made to return a string vs using fmt.Printf.
```

Comment:
This comment seems irrelevant

Reply:
You are correct, the original version did not pass Travis so I converted it to take a io.Writer, will remove. Thanks.